### PR TITLE
Add Premium DNS support to client

### PR DESCRIPTION
**Pull request by zbrag**
This pull request adds support for https://dm.realtimeregister.com/docs/api/dns/zones

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ This SDK currently supports these APIs:
 * [Providers API](https://dm.realtimeregister.com/docs/api/providers)
 * [TLDs API](https://dm.realtimeregister.com/docs/api/tlds)
 * [Processes API](https://dm.realtimeregister.com/docs/api/processes)
+* [Zones API](https://dm.realtimeregister.com/docs/api/zones)
```
Comment (nawaik) [inline README.md:22]:
```suggestion
* [Zones API](https://dm.realtimeregister.com/docs/api/dns/zones)
```

File: src/Api/ZonesApi.php [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        }
+
+        $response = $this->client->get('v2/dns/zones', $query);
+        return DomainZoneCollection::fromArray($response->json());
+    }
+
+    /** @see https://dm.realtimeregister.com/docs/api/dns/zones/create */
+    public function create(
+        string $name,
+        ?string $service = null,
```
Comment (nawaik) [inline src/Api/ZonesApi.php:47]: Do we want to restrict this to DomainZoneServiceEnum?

**Review by nawaik — APPROVED** [review]
Looks good, I have checked all the api's with the documentation and that seems all good. All tested as well.

File: src/Api/ZonesApi.php (hunk 2 above)
Comment (glenn1001) [reply src/Api/ZonesApi.php:47]: I would suggest not creating an enum for service. Current possible values are BASIC and PREMIUM, but I can easily see that types will be added to this list, whenever marketing comes up with a new product.
Not that that would be a huge problem, but that would mean with every type being added, that this client needs to be updated and in the meantime the client can't be used for these new types.

**Review by glenn1001 — APPROVED** [review]

**Comment by zbrag** [comment]
I'm closing this one because of https://github.com/sandwave-io/realtimeregister-php/pull/79 .

**Pull request closed without merging**